grails-app/views/notify/clarificationRequest.gsp

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        User ${user.displayName}, has requested a clarification  for ${issue.typeLabel} ${issue.projectKey}
+    </p>
+    <p>Clarification :</p>
+    <p style="margin-left: 50px;">${raw(comment)}</p>
+    <p>
+        Details regarding this submission can be accessed at:
+        <a href="${issueLink}">${issueLink}</a>
+    </p>
+</div>
```

Comment:
Just a note that I'm glad this pattern met the need for this feature. In retrospect, we could have done something similar to what Consent does with Freemarker templates. In the long run, we probably do want to get away from using GSP files completely, but for now, this is great.